CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 As a minor extension, we also keep a semantic version for the `UNRELEASED`
 changes.
 
+## [0.10.0] UNRELEASED
+
+- Hydra node can now parse `Mainnet` network command line argument.
```

Comment:
Please also bump all the versions in `.cabal` files for packages you touched in this PR to `0.10.0`